aten/src/ATen/native/EmbeddingBag.cpp

```diff
@@ -41,12 +41,14 @@ static void make_offset2bag(const Tensor &offsets, Tensor& offset2bag) {
 
 namespace {
 
-bool isFastPathIndexSelect(const Tensor& src, Tensor& output) {
-  return src.scalar_type() == kFloat && src.stride(1) == 1 && output.stride(1) == 1;
+template<typename index_t>
```

Comment:
I realize this is an existing issue, but a one-line comment explaining this function and its intended use would be helpful